### PR TITLE
Fix typos found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**

### CODE REVIEW

**Review by alexisthual — APPROVED** [review]
LGTM! Thanks :)
Maybe we could add `codespell` to the pre-commit hooks (in this PR or in a different one).

File: examples/00_basics/plot_2_2_coarse_to_fine.py [hunk 1]
```diff
@@ -89,7 +89,7 @@
 # which we sample randomly in this example.
 # You should carefully set the source and target ``selection_radius``
 # as they will greatly affect the sparsity of the computed mappings.
-# They should usally be set using domain knowledge related to the distributions
+# They should usually be set using domain knowledge related to the distributions
 # you are trying to align.
```
Comment (alexisthual) [inline examples/00_basics/plot_2_2_coarse_to_fine.py:93]:
```suggestion
# They should usually be set using domain knowledge related to the
# distributions you are trying to align.
```

**Review by alexisthual — COMMENTED** [review]
One minor flake8 error to fix

**Comment by DimitriPapadopoulos** [comment]
I might add codespell to pre-commit in a different PR.

Right now I have lots if issues with pre-commit, with strange failures on both Ubuntu 18.04 and 22.04. I need to fix them at some point, but right now I cannot user pre-commit.

**Comment by DimitriPapadopoulos** [comment]
I might add codespell to pre-commit in a different PR.

Right now I have lots if issues with pre-commit, with strange failures on both Ubuntu 18.04 and 22.04. I need to fix them at some point, but right now I cannot use pre-commit.

**Comment by DimitriPapadopoulos** [comment]
I might add codespell to pre-commit in a different PR.

Right now I have lots if issues with pre-commit, with strange failures on both Ubuntu 18.04 and 22.04. I need to fix them at some point, but right now I cannot use pre-commit.

**Comment by DimitriPapadopoulos** [comment]
I might add codespell to pre-commit in a different PR.

Right now I have lots if issues with pre-commit, with strange failures on both Ubuntu 18.04 and 22.04. I need to fix them at some point, but right now I cannot use pre-commit.

**Comment by alexisthual** [comment]
Ok, merging!

**Pull request merged**